iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/CatalogConstants.java

```diff
@@ -15,6 +15,8 @@ public final class CatalogConstants {
   /** Used to uniquely identify an update towards a table from user side. */
   public static final String COMMIT_KEY = "commitKey";
 
+  public static final String EVOLVED_SCHEMA_KEY = "evolvedSchema";
```

Comment:
Not for this PR, but do you think we eventually need to bring the evolved schema to top level spec, calling it writerSchema?

Reply:
with this change, the `schema` in the top-level API semantically means `writeSchema`. It's "take this schema as-it-is after this update"

Reply:
I see. then evolved schema key is more for messaging passing in internal table operations.